### ISSUE TITLE
Error with cordova add: Using "requireCordovaModule" to load non-cordova module "elementtree" is not supported.

### ISSUE THREAD

**Issue by kidGodzilla** [huangang/cordova-android-crash-auto-restart#1]
This is informational. When I added this via `config.xml` everything seems to be working fine.

```
cordova plugin add cordova-android-crash-auto-restart
Installing "cordova-android-crash-auto-restart" for android
Failed to install 'cordova-android-crash-auto-restart': CordovaError: Using "requireCordovaModule" to load non-cordova module "elementtree" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
    at Context.requireCordovaModule (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/Context.js:57:15)
    at module.exports (/Users/jamesfuthey/displayjoy-app-cordova/plugins/cordova-android-crash-auto-restart/scripts/install.js:5:22)
    at runScriptViaModuleLoader (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:181:32)
    at runScript (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:157:16)
    at /usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:125:20
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Using "requireCordovaModule" to load non-cordova module "elementtree" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
```

I'm on `9.0.0 (cordova-lib@9.0.1)` if that matters.

Nice plugin by the way!

**Comment by huangang** [comment]
fix in version 0.0.3, you can try it

**Comment by kidGodzilla** [comment]
Thanks!
